common/ledger/util/leveldbhelper/leveldb_helper_test.go

```diff
@@ -157,6 +157,30 @@ func TestFileLock(t *testing.T) {
 	require.NoError(t, os.RemoveAll(fileLockPath))
 }
 
+func TestFileLockLockUnlockLock(t *testing.T) {
+	// create an open lock
+	lockPath := testDBPath + "/fileLock"
+	lock := NewFileLock(lockPath)
```

Comment:
The dir `lockPath` not cleaned up after the test.  In a `defer` you can release the lock and delete the dir